src/util/psbt/map/global.rs

```diff
@@ -13,6 +13,7 @@
 //
 
 use std::collections::BTreeMap;
+use std::collections::btree_map::Entry;
```

Comment:
Why do you import the Entry type here but not in the previous file?

Reply:
Nor in the next it seems.

Reply:
When it was part of a macro I used the full path as we usually do in macros, when it wasn't part of a macro I imported it for better readability.

Reply:
ah ok, that makes perfect sense